templates/ep19/bs/conversations/staff_thread_view.html

```diff
@@ -0,0 +1,128 @@
+{% extends "ep19/bs/conversations/base.html" %}
+
+{% load crispy_forms_tags %}
+
+{% block staff_content %}
+<style type="text/css">
+/* temporary style for readability */
+div.message {
+    border: 1px solid #aaa;
+    margin: 1em 0;
+    padding: 2em;
+    border-radius: 7px;
+}
+
+div.message.user_question {
+    margin-right: 5em;
+}
+
+div.message.staff_reply {
+    background: #eee;
+    margin-left: 5em;
+}
+div.message.internal_note {
+    background: #f93;
+    padding: 1em;
+}
+
+div.message.public_note {
+    padding: .5em;
+    background: #39f;
+}
+</style>
+
+<h3>{{ thread.title }}</h3>
+
+<div class='mv3'>
+    Status: <span class="status-bg-{{ thread.status }}">{{ thread.get_status_display }}</span>
+
+    <form method='POST' action='.' style='display: inline'>
+        {% csrf_token %}
+        <input type='hidden' name='{{ ThreadActions.change_priority }}'>
+        {{ change_priority_form }}
+        <input type='submit' value='update priority'>
+    </form>
+</div>
+
+
+<p class='mv3'>
+started
+<strong title="{{ thread.created }}">{{ thread.created|timesince }} ago</strong>, last activity
+<em title="{{ thread.last_message_date }}">
+    {{ thread.last_message_date|timesince }} ago
+</em>
+</p>
+
+{% if thread.metadata %}
+    {% include "ep19/bs/conversations/_thread_metadata_as_table.html" with thread=thread %}
+{% endif %}
+
+{% block additional_metadata %}{% endblock %}
+
+
+{% for message in thread.messages.all %}
```

Comment:
I find your lack of indentation disturbing

![](https://media.giphy.com/media/1HPUSulSOHDpe/giphy.gif)